CDCgov/prime-reportstream
# 13136: add human readable condition display to azure events

This PR adds human readable conditions to Azure events rather than just the condition code.

Test Steps:
1. Run the project with Azure AppInsights jar setup
2. Run the smoke tests
3. Check Azure logs to see the new condition display present

## Changes
- Added `ConditionSummary` data class
- Added `AzureEventUtils` to handle packaging up Conditions into a form Azure can handle
- Refactored some bundle helper functions

## Checklist

### Testing

## Linked Issues
- Fixes #13378
